core/src/main/java/com/github/games647/fastlogin/core/shared/FloodgateManagement.java

```diff
@@ -120,6 +120,15 @@ public void run() {
 
     }
 
+    /**
+     * Decude if the player can be auto registered.
+     * The config option 'non-conflicting' is ignored by this function.
+     * @return true if the Player can be registered automatically
+     */
+    private boolean isAutoRegisterNotAllowed() {
```

Comment:
Checking for the not allowed seems to be a bit complicated. I think it's better to have `!isAllowed...` or what do you think?

Reply:
It's a lot better that way.